src/parser/sparqlParser/SparqlQleverVisitor.h

```diff
@@ -124,6 +130,11 @@ class SparqlQleverVisitor {
     isInsideConstructTriples_ = true;
   }
 
+  void setStartTime() {
+    startTime_ = absl::FormatTime("%Y-%m-%dT%H:%M:%E3S%Ez", absl::Now(),
+                                  absl::LocalTimeZone());
+  }
```

Comment:
Can you move the implementation to the `.cpp`file. (See what I wrote about the includes above)

Reply:
Moved them to `.cpp`, some includes where also redudant.